common/traits/wc_race_traits.txt

```diff
@@ -1974,6 +1975,158 @@ creature_ancient_wind = {
 	immortal = yes
 	cannot_marry = yes
 
+	random = no
+	customizer = no
+}
+# Pandaria
+creature_pandaren = {
+	diplomacy = 2
+	combat_rating = 5 	# Martial artists
+	
+	ai_ambition = -5 	# Peaceful
+
+	same_opinion = 10
+	hozen_opinion = 10
+	jinyu_opinion = 10
+	grummle_opinion = 10
+	general_opinion = -10
```

Comment:
First, it's not localized.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/86514925-86556700-be26-11ea-8806-33d964822a52.png)

</details>
Second, these opinions don't work.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/86514992-0e3b7100-be27-11ea-9276-399747da161c.png)

</details>
Third, jinyu have +10 opinion in sum of other jinyu.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/86515046-5eb2ce80-be27-11ea-866b-5e7d9bd708c4.png)

</details>